sample-app/src/components/VisualAutocomplete/VisualSearchBar.tsx

```diff
@@ -106,7 +107,6 @@ export default function VisualSearchBar({
           onSubmit={executeQuery}
           onInputChange={value => {
             answersActions.setQuery(value);
-            updateEntityPreviews(value);
```

Comment:
while debugging why the "rose" query wouldn't show any entity previews, I noticed updateEntityPreviews would fire twice for every key stroke. This is because both `onInputChange` and `onInputFocus` are fired whenever somebody types into InputDropdown. entity previews are debounced so you wouldn't notice this just by looking at the networks tab